src/world-company-remuneration/salary-calculator-service-impl.ts

```diff
@@ -3,6 +3,20 @@ const K = "K";
  * @author Jack Napier <jnapier@therealjoker.com>
  */
 export function salary(a: number, b: number) {
+    if (a === null){
```

Comment:
```suggestion
    if (isNewComers){
```
function isNewComers(employeeId: number | null) {
 return employeeId === null
}